X9.146/README.md

```diff
@@ -0,0 +1,406 @@
+# X9.146 Examples
+
+This README file explains how to setup various demos for showing our X9.146
+features in action. X9.146 is a specification of a certificate format that
+allows for dual public keys and signatures in a single certificate.
+
+Traditionally, there are only public key, signature algoirthm specifier and
+signature value. These are known as the native elements. The X9.146 scheme also
+allows for additional alternative public key, signature algorithm specifier and
+signature value as optional X.509 certificate extensions.
+
+The X9.146 specification also specifies how to use these certificates in TLS
+1.3. In the ClientHello message, a CKS extension is added. This extension
+specifies the ability and preference for which signature(s) is/are sent in the
+CertificateVerify message. The presence of the value specifies ability; the
+order of the values specifies preference. The following values are defined: 
+
+- NATIVE 0x01
+- ALTERNATIVE 0x02
+- BOTH 0x03
+- EXTERNAL 0x04 (not supported)
+
+The ServerHello message would have the extension and it would only have a single
+value which would be one of the ones in the list sent over by the client.  That
+is going to specify what is sent in the CertificateVerify message. BOTH is simply the concatenation of the native and alternative signatures; native first.
+
+## Post-Quantum
+
+Tested with these wolfSSL build options:
+
+```sh
+./autogen.sh  # If cloned from GitHub
+./configure --enable-x9-146 --with-liboqs --enable-debug

```

Comment:
please double check config enable options

Reply:
Fixed here and below.